data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
I do agree with @wgrriffel in this point, if you can break the system/functionality changing the script, then it is better to keep it at sources, but again, it is debatable in some cases.